stake-pool/js/package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "stake-pool-js",
+  "version": "0.0.1",
+  "description": "JS bindings for the stake pool program",
+  "private": true,
+  "scripts": {
+    "lint": "npx prettier src/*.ts -w",
+    "compile": "tsc",
+    "test": "./node_modules/mocha/bin/mocha -p ./dist"
```

We should be able to just call `mocha`, but this probably requires a `build` step beforehand
```suggestion
    "test": "mocha -p ./dist"
```